lib/slave/driver/src/nuttx/SPI.cc

```diff
@@ -1,13 +1,19 @@
 #include "kickcat/nuttx/SPI.h"
 
 #include <fcntl.h>
+#include <string>
 
 namespace kickcat
 {
+    SPI:: SPI(int spi_number): SPI_number(spi_number)
```

Comment:
Prefer {}

https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#es23-prefer-the--initializer-syntax